git-issue.sh

```diff
@@ -56,6 +56,66 @@ else
   DATEBIN="date"
 fi
 
+
+##
+## PLUGIN AREA
+## (how to specify in different file when run from different location)
+##
+
+opt_archive=1
+opt_sign=1
+
+sig_by_state() {
```

Comment:
Please add a comment describing what this function does. Consider also expanding `sig`.